src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -144,21 +146,22 @@
   }
 
   private static String getGapicVersion() {
-    try {
-      Properties properties = new Properties();
-      String mavenPropertiesPath = META_FILE_ROOT
-          + GROUP_ID + "/"
-          + ARTIFACT_ID + "/pom.properties";
-      InputStream inputStream =
-          ImageAnnotatorSettings.class.getResourceAsStream(mavenPropertiesPath);
-      if (inputStream != null) {
-        properties.load(inputStream);
-        return properties.getProperty(META_VERSION_KEY, "");
+    if (gapicVersion == null) {
+      try {
+        Properties properties = new Properties();
```

Comment:
Maybe not necessary in this PR, but it feels like the loading logic here could be handled by generalizing GaxPropertiesProvider (https://github.com/googleapis/gax-java/blob/master/src/main/java/com/google/api/gax/core/GaxPropertiesProvider.java)

That way we have less logic in generated code.